### DISCUSSION TITLE
public key and pool key ?

### DISCUSSION THREAD

**Discussion by yanadhorn** [swar/Swar-Chia-Plot-Manager#146] [Q&A]
if i run the manager on the same PC that i have original chia install on it. do i have to fill those two value in configuration.

**Comment by ducnv1406** [comment]
If you have only one fingerprint on chia, I think it's OK

But my case:
- I have 2 fingerprint on my CHIA app
- I leave it blank in SWAR config, but I check the log file

public key: 907307...eeb76a 
farmer public key: 9919b7...cba[0m

while my 2 public key is
83f1ec...a6e75c
ad20b8...bc19b4

Pls help explain
